.github/.github/workflows/mark_stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Mark Inactive Issues and PRs
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v5
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "This issue will be closed due to inactivity."
+          stale-pr-message: "This PR will be closed due to inactivity."
```

Comment:
Same for pr message

`This PR has been inactive for 30 days. If it's waiting for a review, please reach out to the team. Otherwise, please update the PR or it will be closed due to inactivity.`